### ISSUE TITLE
The method to meansure FLOPs

### ISSUE THREAD

**Issue by zeyuliu1037** [jyjohnchoi/SMoP#3]
Hi, thank you for your great work!

Can you share the specific hyper-parameters for each subtask in the SuperGLUE benchmark? We try some combinations of the learning rate and total training epoch from Appendix C, but we find the performance gap for some tasks like CB is very high.

BTW, I am trying to measure the FLOPs by using the DeepSpeed library. However, I noticed that the FLOPs for each method are different. Can you tell me what specific things I need to pay attention to when measuring FLOPs?
